tony-examples/tony-in-gcp/README.md

```diff
@@ -219,15 +219,93 @@ You can also track Job status from DataProc Jobs tab: Cloud Console -> Big Data
 
 ![img](https://i.imgur.com/3XgZI5Z.png)
 
+## Autoscaling
+
+
+Estimating the "right" number of cluster workers (nodes) for a workload is difficult, and a single cluster size for an entire pipeline is often not ideal. User-initiated Cluster Scaling partially addresses this challenge, but requires monitoring cluster utilization and manual intervention.
+
+The Cloud Dataproc AutoscalingPolicies API provides a mechanism for automating cluster resource management and enables cluster autoscaling. An Autoscaling Policy is a reusable configuration that describes how clusters using the autoscaling policy should scale. It defines scaling boundaries, frequency, and aggressiveness to provide fine-grained control over cluster resources throughout cluster lifetime.
+
+Please find a sample below:

```

Comment:
if I'm the user, it would be helpful to have a quick gif or video on this

Reply:
Gif added